app/pathogen/arbovirus/dashboard/(map)/arbo-map-cluster-utils.tsx

```diff
@@ -0,0 +1,225 @@
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem; padding-left: 5px; border-left: 5px solid ${pathogenColors["YF"]}">YF</div>`;
+  if (props.properties.WNV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem; padding-left: 5px; border-left: 5px solid ${pathogenColors["WNV"]}">WNV</div>`;
+  popupHTML += `</div>
+            <div style="display: flex; flex-direction: column;">`;
+
+  if (props.properties.ZIKV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem;">${props.properties.ZIKV}</div>`;
+  if (props.properties.CHIKV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem;">${props.properties.CHIKV}</div>`;
+  if (props.properties.DENV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem;">${props.properties.DENV}</div>`;
+  if (props.properties.MAYV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem;">${props.properties.MAYV}</div>`;
+  if (props.properties.YF > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem;">${props.properties.YF}</div>`;
+  if (props.properties.WNV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem;">${props.properties.WNV}</div>`;
+
+  popupHTML += `</div>
+        </div>
+        <div style="font-size: 0.875rem; font-weight: 300; text: center">Click to zoom in</div>
+
+    </div>`;
+
+  // Cannot use the react-gl Popup method as we need to use useRef to access the underlying method
+  // and we cannot do that when these popups are loaded dynamically and not statically.
+  const popup = new mapboxgl.Popup({
+    offset: [0, -7],
+    closeOnClick: false,
+  });
+
+  const onMouseEnter = (event: any) => {
+    event.target.style.cursor = "zoom-in";
+    popup.setLngLat(props.coords).setHTML(popupHTML).addTo(props.map);
+  };
+
+  const onMouseLeave = (event: any) => {
+    event.target.style.cursor = "";
+    popup.remove();
+  };
+
+  return (
+    <svg
+      width={piChartDiameter}
+      height={piChartDiameter}
+      viewBox={`0 0 ${piChartDiameter} ${piChartDiameter}`}
+      text-anchor="middle"
+      style={{ font: `${fontSize}px sans-serif`, display: "block" }}
+      onMouseEnter={onMouseEnter}
+      onMouseLeave={onMouseLeave}
+      onClick={() => {popup.remove()}}
+    >
+      {counts.map((count, i) =>
+        donutSegment(
+          offsets[i] / total,
+          (offsets[i] + count) / total,
+          piChartOuterRadius,
+          piChartInnerRadius,
+          pathogenColors[arboColorNames[i]]
+        )
+      )}
+      <circle cx={piChartOuterRadius} cy={piChartOuterRadius} r={piChartInnerRadius} fill="white" />
+      <text dominant-baseline="central" transform={`translate(${piChartOuterRadius}, ${piChartOuterRadius})`}>
```

Comment:
```suggestion
      <text dominantBaseline="central" transform={`translate(${piChartOuterRadius}, ${piChartOuterRadius})`}>
```
Is the correct name for this property, we are getting console errors because of it.